<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info_file_sys</class>
 <widget class="QDialog" name="info_file_sys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Информация о файловой системе</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 243, 250);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string>1. Вывод информации о логических дисках, именах, метке тома,</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string> размере и типе файловой системы (модуль psutil). </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tE_info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>511</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_refresh">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
